src/libclipper/src/containers.cpp

```diff
@@ -108,8 +113,16 @@ void ActiveContainers::add_container(VersionedModelId model, int connection_id,
                      model.get_name(), model.get_id(), connection_id,
                      replica_id, get_readable_input_type(input_type));
   boost::unique_lock<boost::shared_mutex> l{m_};
+
+  // Set a default batch size of -1
+  int batch_size = -1;
```

Comment:
Define a `constexpr DEFAULT_BATCH_SIZE = -1` constant in `container.hpp` and refer to it in all relevant locations, instead of using the value `-1` directly